bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,16 @@
+
+# bubble sort function
+def bubbleSort(alist):
+    for num in range(len(alist)-1,0,-1):
+        for i in range(num):
+            if alist[i]>alist[i+1]:
+                temp = alist[i]
+                alist[i] = alist[i+1]
+                alist[i+1] = temp
+
+
+# main function 
+def main():
+    alist = [54,26,93,17,77,31,44,55,20]
+    bubbleSort(alist)
+    print(alist)
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp470kal1c/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmp470kal1c/bubble_sort/Python/bubble_sort.py
@@ -13,4 +13,4 @@
 def main():
     alist = [54,26,93,17,77,31,44,55,20]
     bubbleSort(alist)
-    print(alist)+    print(alist)
```